### ISSUE TITLE
How to edit tracker list?

### ISSUE THREAD

**Issue by Young-Lord** [MuntashirAkon/AppManager#664]
**Describe the existing feature/documentation**
In 1-Click Ops Page, Block/Unblock Trackers

**Describe your problem(s)**
I'm using MIUI, it uses Mipush(like FCM) to perform good push experience. However, App Manager blocks "com.xiaomi.mipush" by default. So, how can I edit the trackers list without repacking the app?

**Additional contexts**

**Comment by MuntashirAkon** [comment]
@gnuhead-chieb: What is your input regarding Xiaomi Push Services? You seem to have one of the approvers of the tracker in ETIP.

**Comment by gnuhead-chieb** [comment]
Mi push is useless for people live country they can use Firebase.
so it only violate privacy for most people.
I read website of mipush and it has tracking feature worse than firebase.

but i know it can be useful for people who living place cant use firebase,so allowing excluding tracker to user self is useful for some people.

**Comment by MuntashirAkon** [comment]
> but i know it can be useful for people who living place cant use firebase,so allowing excluding tracker to user self is useful for some people.

But I am sure you know the consequences of adding such a feature. Once we bend our principles, there will be no coming back.

I think it would be far better if we add some sort of include/exclude lists for components in general instead of letting users _edit_ trackers. This way they could also be able to fix certain issues regarding Play Services or add components (such as the Facebook ones) that they don't like but not present in ETIP.

**Comment by Young-Lord** [comment]
Yes, mipush absolutely violate privacy in some way. I agree that include/exclude list is better than editing too. So, is this issue a *feature request* at all?
